mdpow/config.py

```diff
@@ -92,35 +92,42 @@
 
 from __future__ import absolute_import
 
-import os, errno
+from typing import Optional, Dict, List, Any, Union
+import os
+import errno
 from pkg_resources import resource_filename, resource_listdir
 import yaml
 
 import numpy as np
 
 import logging
-logger = logging.getLogger("mdpow.config")
 
+logger = logging.getLogger("mdpow.config")
 
 # Reading of configuration files
 # ------------------------------
 
 #: Locations of default run input files and configurations.
 defaults = {
     "runinput": resource_filename(__name__, "templates/runinput.yml"),
-    }
+}
+
 
-def merge_dicts(user, default):
+# TODO test merge_dicts
```

Comment:
replaced with recursive method for merging dict containing multiple other dict, passed runinput